var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -188,10 +197,11 @@ class Llvm(CMakePackage, CudaPackage):
         description="Build with OpenMP capable thread sanitizer",
     )
     variant(
-        "omp_as_runtime",
-        default=True,
-        when="+clang @12:",
-        description="Build OpenMP runtime via ENABLE_RUNTIME by just-built Clang",
+        "omp",
```

Comment:
Same variant issue as above, name needs to have a deprecation period.

Reply:
Ok. I will then go further and make it `openmp`, `openmp_tsan` and `openmp_debug`.